bin/verify-binary-architecture.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
Regular shell please 🙏

Reply:
Ahh, ok – shall we change it for all the other scripts as well then?

`shell: bash` is being used for all the inline scripts of the workflow steps. That's why I went for `bash` to not mix and match shells.

Reply:
Oh! I'm surprised by this. OK let's stick with bash, good call.